docs/modules/ROOT/pages/explanations/commodore-components.adoc

```diff
@@ -0,0 +1,49 @@
+= Commodore component best practices
+
+== Handling major changes
```

Comment:
In a separate PR or commit, I would also welcome a page about handling Helm charts, or more in general how to better support different Kubernetes distributions.
I'm thinking along the "lookup table" where we configure different YAML blocks based on a single switch.

Reply:
Subpages makes sense I think. In the navigation, we can group them together under best practices, e.g. 
```
Explanation
  Commodore Components Best Practice
    * Secrets
    * Major changes
    ...
```